app/src/main/java/com/github/polypoly/app/WelcomeActivity.kt

```diff
@@ -31,9 +46,84 @@ import com.github.polypoly.app.ui.theme.PolypolyTheme
 class WelcomeActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        val db = Firebase.database
+        remoteDB = RemoteDB(db, "live")
+
+        addFakeDataToDB() //> Uncomment this line if you need to refresh the fake data within the DB
+
         setContent { WelcomeContent() }
     }
 
+    /**
+     * DEBUG function
+     * Add fake data (possibly duplicate from tests' fake data) if the data in DB were corrupted
+     */
+    fun addFakeDataToDB() {
```

Comment:
To we need to keep a debug function into our final code?

Reply:
Now that we are connected to the real database, we need it to be populated with real values so that the application can make sense (especially now that not all feature exist yet). For example, when going to the JoinGameLobby activity,  if no game lobby exist yet in the DB then nothing happen in the activity (all codes fail and no gamelobby is shown in the list of existing ones). The same happens when going to the Profile activity, and this is valid for handmade testing, ongoing development and the live demos.
This is the reason why I added this "debug" function (maybe not the best adjective) to populate the DB at launch to ensure we have interesting data to work with (it may be corrupted between two App launches).
This has the downside of preventing persistency, but unit tests already test that it works and the current app state makes that acceptable.

Reply:
But couldn't we just populate the DB once and then just get elements there?

Reply:
No because they can easily be corrupted through wrong manipulation (I personally experienced it a few time in this sprint 😢) and prevent repeatability when developing new features. Moreover as of today we all use the same DB, so data can cross over if 2 people successively their code which may lead to issues. For example is a class gets new fields in the code of 1 person, if an other person run his code later than the data will be corrupted on his point of view: refreshing the data every time solves that.
I do not believe this should necessarily stay for a lot of sprints, but at least for the next one this should come be useful.